main/zoe/guide/price-authority.md

```diff
@@ -46,6 +46,16 @@ This means that the `priceAuthority` asserts that when `timestamp` according to
 `amountIn` and `amountOut` are ERTP amounts for the `brandIn` and `brandOut` you
 requested.
 
+## Mutable price quotes
+
+A `PriceQuote` is a record (has fields rather than methods), while `MutableQuote` is
+an object (methods, no fields). `MutableQuote`'s method `getPromise(`) returns 
+a `Promise` for a `PriceQuote`, which is the same `Promise` returned by the `quoteWhenLTE()`
+API method and its siblings. 
```

Comment:
Sorry, these words were me explaining to you, not me providing wording for the doc. I don't think the comparison to `PriceQuote` as a record belongs here.
```suggestion
A `MutableQuote` is an object. `MutableQuote`'s method `getPromise(`) returns 
a `Promise` for a `PriceQuote`, which serves the same purpose as the return value from `quoteWhenLTE()` etc. 
```

Reply:
I dunno. I think this might be significant enough to be worth including to make sure they realize this, as it affects how they think of/use the two? At the very least, I don't see any harm in including it, and it might do some good?

Reply:
The description of priceQuote as a record is redundant with the paragraph just above: "Receiving a quote".  I'd prefer "is a record with an amount and a payment" over " is an amount, payment pair".  This section is about MutableQuotes, which are a different way of getting a priceQuote. It might make more sense to say that you can use quoteWhenGT and its ilk to get a priceQuote directly, or mutableQuoteWhenGT to get a reusable object.